test/integration/patientInvoice.js

```diff
@@ -69,7 +68,7 @@ describe('(/invoices) Patient Invoices', function () {
     it('GET /invoices/ should return all invoices if no query string provided', function () {
       return agent.get('/invoices')
         .then((res) => {
-          helpers.api.listed(res, numInvoices + numCreatedInvoices);
+          helpers.api.listed(res, numInvoices + numCreatedInvoices - 1);
```

Comment:
Why?

Reply:
One was deleted.  I supposed it could be in a removedInvoices variable.  I'll do that.